fixity/fixity.py

```diff
@@ -48,6 +48,8 @@ def fetch_environment_variables(namespace):
     namespace.ss_url = _get_environment_variable('STORAGE_SERVICE_URL')
     if not namespace.ss_url.endswith('/'):
         namespace.ss_url = namespace.ss_url + '/'
+    namespace.ss_user = _get_environment_variable('STORAGE_SERVICE_USER')
+    namespace.ss_key = _get_environment_variable('STORAGE_SERVICE_KEY')
```

Comment:
I think we can leave these as required (because they are mandatory for SS 0.8). In exchange, I would suggest bumping fixity's version from 0.1 to 0.2, as a way to differentiate the SS version it supports ( so fixity==0.1 would support SS<=0.7 or less, fixity==0.2 would support SS==0.8 )